packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "d5746045ac740d5f28f238dc55ec95c89d2b7cd9",
-    "ref_origin" : "dcos-mesos-1.0.x-837bb4e"
+    "ref": "c8e8c70a71b2849ee37d3c6b9f396ceaac73369e",
+    "ref_origin" : "dcos-mesos-master-35ce5042"
```

Comment:
If you're going to bump this in dcos master, could you also update https://github.com/dcos/dcos/blob/master/packages/mesos/README.md with the new cherry-pick(s) for 1.1.x / master?